### ISSUE TITLE
List mark-up used unnecessarily

### ISSUE THREAD

**Issue by jrbenn** [logicly-au/pmhc-mds-user-documentation#23]
The "Edit on GitHub" link does not have any related item to be marked-up within a list structure. The unordered list mark-up is used unnecessarily for the link. As a result, screen reader users find it difficult to understand the information effectively.

Issue # 20 in https://docs.google.com/spreadsheets/d/13iTb4aW7KulDcbj5Q2xD78H4KOrMmhvfIDW1JMffY7o/edit?gid=155984173#gid=155984173
